src/main/java/org/spburegistry/backend/ExceptionHandler/GlobalExceptionHandler.java

```diff
@@ -5,15 +5,15 @@
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.ControllerAdvice;
 import org.springframework.web.bind.annotation.ExceptionHandler;
-import org.spburegistry.backend.ExceptionHandler.exception.NoSuchUserException;
+import org.spburegistry.backend.ExceptionHandler.exception.NoSuchEntityException;
 
 import lombok.extern.slf4j.Slf4j;
 
 @ControllerAdvice
 @Slf4j
 public class GlobalExceptionHandler {
     @ExceptionHandler
-    public ResponseEntity<AppError> catchNoSuchUserException(NoSuchUserException e) {
+    public ResponseEntity<AppError> catchNoSuchUserException(NoSuchEntityException e) {
         log.error(e.getMessage(), e);
         return new ResponseEntity<>(new AppError(HttpStatus.NOT_FOUND.value(), e.getMessage()), HttpStatus.NOT_FOUND);
     }
```

Comment:
I think it is needed to add abstract exception handler to handle any unexpected errors

Reply:
Done